build/messages/src/Messages.json

```diff
@@ -344,7 +344,7 @@
     "description": "No systems"
   },
   "notMember": {
-    "defaultMessage": "Not a member? Start a trial!",
+    "defaultMessage": "Don't have a RHEL subscription? Start a trial.",
```

Comment:
This taken care of with the merged PR 
https://github.com/RedHatInsights/insights-dashboard/pull/417/files